### ISSUE TITLE
'Back to top' button incorrectly positioned when using the filter on the jobs page in mobile

### ISSUE THREAD

**Issue by CheungJ** [smashingmagazine/redesign#27]
**- Do you want to request a *feature* or report a *bug*?**
Bug

**- What is the current behavior?**
'Back to top' button incorrectly positioned when using the filter on the jobs page in mobile. 

**- If the current behavior is a bug, please provide the steps to reproduce.**
1. Open the jobs page on mobile 
2. Scroll down, so the filters section sticks to the bottom of the screen 
3. Click on the 'filter' button to expand the filters

The 'back to top' button is raised higher than it should be. 

**- What is the expected behavior?**
The 'back to top' button should be nearer to the filters section.  

**- Please mention your operating system version and the version of your browser**
OnePlus 3T, OxygenOS 4.0.3, Android version 7, Google Chrome 56.0.2924.87

**Comment by vitalyfriedman** [comment]
Indeed, we haven't spent nearly as much time on Jobs as we did on the other pages, so we'll be looking into these issues in detail for sure. Thanks, @CheungJ!

**Comment by CheungJ** [comment]
👍

**Comment by vitalyfriedman** [comment]
Still on our to-do list ;-) Jobs will be addressed after the magazine + checkout + membership are finished.
